Products/ZenUI3/browser/resources/js/zenoss/HierarchyTreePanel.js

```diff
@@ -406,16 +406,6 @@
             var store =  this.getStore();
             Zenoss.HierarchyTreePanel.superclass.initEvents.call(this);
 
-            if (this.selectRootOnLoad && !Ext.History.getToken()) {
```

Comment:
we should listen store load event instead of rootNode expand to select selectRootOnLoad;